Userland/Libraries/LibWeb/Animations/Animatable.cpp

```diff
@@ -150,8 +150,13 @@ void Animatable::remove_transition(CSS::PropertyID property_id)
 
 void Animatable::clear_transitions()
 {
-    m_transition_attribute_indices.clear();
-    m_transition_attributes.clear();
+    for (auto& property_id : m_associated_transitions.keys()) {
+        if (m_associated_transitions.contains(property_id)) {
+            m_associated_transitions.clear();
+            m_transition_attribute_indices.clear();
+            m_transition_attributes.clear();
+        }
+    }
```

Comment:
There's no need to put the `clear()` invocation inside a for loop, it just needs to be called once:
```suggestion
    m_transition_attribute_indices.clear();
    m_transition_attributes.clear();
    m_associated_transitions.clear();
```